bitnami/apisix/templates/data-plane/vpa.yaml

```
@@ -0,0 +1,46 @@
+{{- if and (.Capabilities.APIVersions.Has "autoscaling.k8s.io/v1/VerticalPodAutoscaler") .Values.dataPlane.autoscaling.vpa.enabled }}
+apiVersion: {{ include "common.capabilities.vpa.apiVersion" . }}
+kind: VerticalPodAutoscaler
+metadata:
+  name: {{ include "apisix.data-plane.fullname" . }}
+  namespace: {{ include "common.names.namespace" . | quote }}
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    app.kubernetes.io/part-of: apisix
+    app.kubernetes.io/component: data-plane
+   {{- if .Values.commonLabels }}
+   {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
+   {{- end }}
+  annotations:
+  {{- if .Values.commonAnnotations }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
+  {{- end }}
+  {{- if .Values.dataPlane.autoscaling.vpa.annotations }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.dataPlane.autoscaling.vpa.annotations "context" $ ) | nindent 4 }}
+  {{- end }}
+spec:
+  resourcePolicy:
+    containerPolicies:
+    - containerName: apisix
+      {{- with .Values.dataPlane.autoscaling.vpa.dataledResources }}
```

This is a typo, probably caused by string substitution
```suggestion
      {{- with .Values.dataPlane.autoscaling.vpa.controlledResources }}
```